src/EditorFeatures/Test2/FindReferences/FindReferencesTests.OperatorSymbols.vb

```diff
@@ -186,6 +186,48 @@ Class A
 End Class
         </Document>
     </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WorkItem(30642, "https://github.com/dotnet/roslyn/issues/30642")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestCSharpFindReferencesOnBuiltInOperatorWithUserDefinedEquivalent() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+class A
+{
+    void Goo(string a, string b, int x, int y)
+    {
+        var m = a $$[|==|] b;
+        var n = a [|==|] b;
+        var o = x == y;
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WorkItem(30642, "https://github.com/dotnet/roslyn/issues/30642")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestVisualBasicFindReferencesOnBuiltInOperatorWithUserDefinedEquivalent() As Task
```

Comment:
Should this also test cross language?

Reply:
I can add cross language tests for this.  Thanks!